### DISCUSSION TITLE
Application Screenshots on macOS - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#71] [General]
# Application Screenshots on macOS - Alan Pope's blog

I initially started typing this as short -[ Contrafibularities ]- segment for my free, weekly newsletter. But it got a bit long, so I broke it out into a blog post instead.
About that newsletter The newsletter is emailed every Friday - subscribe here, and is archived and available via RSS a few days later. I talked a bit about the process of setting up the newsletter on episode 34 of Linux Matters Podcast.

https://popey.com/blog/2024/07/application-screenshots-on-macos/

**Comment by skellat** [comment]
This is amazing.  I'm used to just hitting Shift+Command+4 to get crosshairs to take a screenshot of whatever I select on-screen.

**Reply by popey** [reply]
Yeah, I use Shift+Command+5 usually, as I can never remember which one does what, and 5 allows you to pick.
